src/include/duckdb/parser/keyword_helper.hpp

```diff
@@ -9,13 +9,14 @@
 #pragma once
 
 #include "duckdb/common/common.hpp"
+#include "duckdb/parser/simplified_token.hpp"
 
 namespace duckdb {
 
 class KeywordHelper {
 public:
 	//! Returns true if the given text matches a keyword of the parser
-	static bool IsKeyword(const string &text);
+	static bool IsKeyword(const string &text, const vector<KeywordCategory> &filter_type = {});
```

Comment:
Can we bubble the keyword type up all the way, so that we have `KeywordHelper::KeywordCategory(const string &text)` - and `KeywordHelper::IsKeyword()` is essentially `KeywordHelper::KeywordCategory(text) != KeywordCategory::NONE`

Reply:
Ah, I had understood that you just wanted it out of the third-party